src/chapters/configuring.xml

```diff
@@ -5307,6 +5307,50 @@ guaclog: INFO: All files interpreted successfully.</computeroutput>
                         </listitem>
                     </itemizedlist>
                 </section>
+                <section xml:id="ldap-tokens">
+                    <title>LDAP Extension Tokens</title>
+                    <indexterm>
+                        <primary>tokens</primary>
+                        <secondary>ldap</secondary>
+                    </indexterm>
+                    <para>The LDAP extension will read user attributes provided by the LDAP
+                        server and specified in the <filename>guacamole.properties</filename>
+                        file.  The attributes retrieved for a user are configured using the
+                        <param>ldap-user-attributes</param> parameter.  The user must be able
+                        to read the attribute values from their own LDAP object.  The token
+                        name will be prepended with the <constant>LDAP_</constant> prefix.  As
+                        an example, configuring the following line in
+                        <filename>guacamole.properties</filename> file will produce the
+                        below tokens that can be used in connection configurations:</para>
+                    <programlisting>ldap-user-attributes: cn, givenName, sn, mobile, mail</programlisting>
+                    <itemizedlist>
+                        <listitem>
+                            <para>
+                                <varname>${LDAP_CN}</varname>
+                            </para>
+                        </listitem>
+                        <listitem>
+                            <para>
+                                <varname>${CAS_GIVENNAME}</varname>
```

Comment:
Copypasta - this and a few others have a `CAS_` prefix.

Reply:
Oops, fixed.